src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -310,6 +310,7 @@ Apply to thead to make headers not bold.
 
 .text-preserve-space {
     white-space: pre-wrap;
+    word-break: break-all;
```

Comment:
There is also a `word-wrap` property which seems to address this issue better (smarter break). However, browser support is not that well. 

Let me look around a little bit more. Thanks for the inputs @YongJieYongJie